src/vs/vscode.proposed.d.ts

```diff
@@ -625,6 +625,11 @@ declare module 'vscode' {
 		 * An [event](#Event) that fires when the log level has changed.
 		 */
 		export const onDidChangeLogLevel: Event<LogLevel>;
+
+		/**
+		 * The url protocol of the editor, like 'vscode', 'vscode-insiders'.
+		 */
+		export const urlProtocol: string;
```

Comment:
Not sure, the thing is called ur**i**-handler, now we talk about ur**l** protocol. What's correct? Why being inconsistent? Also protocol vs scheme?

Reply:
We should also update this https://github.com/Microsoft/vscode/blob/a3fc8198a7a274b580e8fbf7664e13a8bd8df647/src/vs/vscode.d.ts#L6643

Reply:
We should call it `uriScheme` as `uriHandler` stands for the application.